src/applications/vaos/tests/project-cheetah/components/NewBookingPage.unit.spec.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { expect } from 'chai';
+import { mockFetch, resetFetch } from 'platform/testing/unit/helpers';
+import { createTestStore, renderWithStoreAndRouter } from '../../mocks/setup';
+import { NewBooking } from '../../../project-cheetah';
+import { getDirectBookingEligibilityCriteriaMock } from '../../../tests/mocks/v0';
+import { mockDirectBookingEligibilityCriteria } from '../../../tests/mocks/helpers';
+
+const initialState = {
+  featureToggles: {
+    vaOnlineSchedulingDirect: true,
+  },
+  user: {
+    profile: {
+      facilities: [
+        { facilityId: '983', isCerner: false },
+        { facilityId: '984', isCerner: false },
+      ],
+    },
+  },
+};
+
+describe('VAOS <ConfirmedAppointmentDetailsPage>', () => {
+  beforeEach(() => {
+    mockFetch();
+  });
+
+  afterEach(() => {
+    resetFetch();
+  });
+
+  it('should not redirect the user to the Contact Facilities page', async () => {
+    const store = createTestStore({
+      ...initialState,
+    });
+
+    const response = getDirectBookingEligibilityCriteriaMock();
```

Comment:
You could also do `getDirectBookingEligibilityCriteriaMock({ typeOfCareId: '301' })`, and get rid of the next two lines.